doc/administrator/editing.rst

```diff
@@ -76,6 +78,45 @@ corresponding to this field is *Related Postgres table* in the admin interface.
 
     It is recommended to limit the editing to 30 editable layers per user.
 
+Configure layer editing
+-----------------------
+An overview of the metadata for the configuration of the editing can be found here:
+:ref:`administrator_administrate_metadata`.
+
+Registering edit time and user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* To register the time of the change of an edited item, a column of type
+  ``timestamp without timezone`` is needed in the table of the layer. In
```

Comment:
Are you sure about this? shouldn't be `timestamp with timezone`?

Reply:
As I see, `timestamp with timezone` is not supported as the setted value is not timezone aware : https://github.com/camptocamp/c2cgeoportal/blob/c87b857bc2f08051ca514e3a37b2c32099e71ed5/geoportal/c2cgeoportal_geoportal/views/layers.py#L384

Reply:
To be able to use a `timestamp with timezone` we would need `datetime.now(timezone.utc)`